### PR TITLE
1085 dsr modal

### PR DESCRIPTION
**Pull request by momimomo**

### CODE REVIEW

File: frontend/src/components/eois/details/headerOptions/agencyDirectHeaderOptions.js [hunk 1]
```diff
@@ -114,13 +118,14 @@ class AgencyDirectHeaderOptions extends Component {
       },
     ];
 
-    if ((!isCompleted && isPublished && this.isActionAllowed(hasEditPublishedPermission)) ||
-    (!isPublished && status === 'Sen' && ((hasEditSentPermission && isAdvEd && isFocalPoint)
-            || (!isCompleted && hasEditPublishedPermission && isMFT && isFocalPoint)))) {
+    console.log(isCompleted, isPublished, hasEditPublishedPermission);
```
Comment (kenzo23312) [inline frontend/src/components/eois/details/headerOptions/agencyDirectHeaderOptions.js]: remove log

File: frontend/src/components/eois/modals/editDsr/editDsrModal.js [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+import { submit } from 'redux-form';
+import Grid from 'material-ui/Grid';
+import ControlledModal from '../../../common/modals/controlledModal';
+import EditDsrForm from './editDsrForm';
+import { updateCfei } from '../../../../reducers/newCfei';
+import { errorToBeAdded } from '../../../../reducers/errorReducer';
+import { selectCountriesWithOptionalLocations } from '../../../../store';
+
+const messages = {
+  title: 'Create new direct selection/retention',
```
Comment (kenzo23312) [inline frontend/src/components/eois/modals/editDsr/editDsrModal.js]: Edit direct sele...

File: frontend/src/components/eois/modals/editDsr/editDsrModal.js [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+  save: 'Save',
+  error: 'Unable to create new direct selection/retention',
```
Comment (kenzo23312) [inline frontend/src/components/eois/modals/editDsr/editDsrModal.js]: 'to update'

File: frontend/src/components/eois/modals/editDsr/editDsrModal.js [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+      disabled: false };
+    this.handleSubmit = this.handleSubmit.bind(this);
+    this.handleDialogSubmit = this.handleDialogSubmit.bind(this);
+    this.handleConfirmation = this.handleConfirmation.bind(this);
+  }
+
+  handleSubmit(values) {
+    const normalizedValues = values;
+    normalizedValues.locations = [];
+    const i = values.countries;
```
[the remark this answers is not in the document]
Comment (momimomo) [reply frontend/src/components/eois/modals/editDsr/editDsrModal.js]: Direct assignment of function parameter is considered an error by es-lint:
``Assignment to property of function parameter "values". (no-param-reassigns)``

**Pull request merged**